ibacm/src/acm.c

```diff
@@ -2016,7 +2016,7 @@ acm_ep_insert_addr(struct acmc_ep *ep, const char *name, uint8_t *addr,
 		return EINVAL;
 
 	memset(tmp, 0, sizeof tmp);
-	memcpy(tmp, addr, addr_len);
+	memcpy(tmp, addr, acm_addr_len(addr_type));
```

Comment:
It is really strange to pass in addr_len and not use it. Are you sure the bug isn't that the caller is using the wrong addr_len for the address type?

Otherwise I suggest deleting addr_len as a parameter as well

Reply:
Yes, I agree. An oversight from me. Just pushed a v2 where the addr_len is re-factored out.

Reply:
I do see a s/form/from/ fix. You want me to fix it?